### ISSUE TITLE
[web] Semantic tree traversal order

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#45206]
Today in flutter for web, traversing order of nodes are not always correct. For example a11y focus shifts to the next tab before it goes to the contents of the tab (and there is no way of accessing the content of the tabs without jumping on all tabs.) 

Traversal should have a logical order. 

role="group" can be used to group the elements.

**Comment by yjbanov** [comment]
This is fixed by https://github.com/flutter/engine/pull/32712. Forgot to close it.
